features/bevragen/zoek-met-geslachtsnaam-en-geboortedatum/dev/gba.feature

```diff
@@ -0,0 +1,71 @@
+#language: nl
+
+@gba
+Functionaliteit: Zoek met geslachtsnaam en geboortedatum (GBA)
+
+Rule: voorvoegsel is niet hoofdlettergevoelig
+
+  Scenario: Zoek met volledige voorvoegsel in hoofdletters
```

Comment:
Heeft het nog toegevoegde waarde om ook nog versie's te maken met 'van' en 'Van' (of zelfs 'vAn')?